schedule/__init__.py

```diff
@@ -333,24 +333,36 @@ def tag(self, *tags):
 
     def at(self, time_str):
         """
-        Schedule the job every day at a specific time.
+        Specify a particular time that the job should be run at.
 
-        Calling this is only valid for jobs scheduled to run
-        every N day(s).
-
-        :param time_str: A string in `XX:YY` format.
+        :param time_str: A string in one of the following formats: `HH:MM:SS`,
+            `HH:MM`,`:MM`, `:SS`.
```

Comment:
Thanks but we'll need to expand the docstring and add a couple of concrete examples for how the `:MM` / `:SS` syntax works. With this PR `at()` is now doing a lot more work and the support burden for this library is already super high (for me personally—getting about 3-5 emails a week or so with questions and I mostly have to ignore them due to time constraints). So it's dear to my heart that the docs are as helpful as we can make them :-)

Reply:
Ok I added two sentences, one about the format needing to match when the job is being repeated, and another about how the difference between `:MM` and `:SS` is detected, and gave two concrete examples.